tests/graph/test_op.py

```diff
@@ -232,3 +232,26 @@ def perform(self, *_):
 
     x = pt.TensorType(dtype="float64", shape=(1,))("x")
     assert SomeOp()(x).type == pt.dvector
+
+
+def test_op_name():
+    x = pt.vector("x")
+    y = pt.vector("y")
+    op_name = "op_name"
+
+    class MultiOutOp(Op):
+        def make_node(self, *inputs):
+            outputs = [pt.dmatrix(), pt.dmatrix()]
+            return Apply(self, list(inputs), outputs)
+
+        def perform(self, node, inputs, outputs):
+            outputs[0] = pt.matrix()
```

Comment:
This is wrong but not needed. You can replace contents of perform with raise NotImplementedError